spa/plugins/bluez5/bluez5-device.c

```diff
@@ -107,9 +160,23 @@ static int emit_nodes(struct impl *this)
 		}
 	}
 
+	printf ("Bluez5 sink nodes emitted\n");
```

Comment:
I think you should be using spa_log* instead of printf (everywhere in this commit)